packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -410,6 +411,37 @@ export const announceClientService = ({
   app.provide('$clientService', clientService)
 }
 
+export const announceArchiverService = ({
+  app,
+  configStore,
+  userStore,
+  capabilityStore
+}: {
+  app: App
+  configStore: ConfigStore
+  userStore: UserStore
+  capabilityStore: CapabilityStore
+}): void => {
+  app.config.globalProperties.$archiverService = new ArchiverService(
+    app.config.globalProperties.$clientService,
+    userStore,
+    configStore.serverUrl,
+    computed(
+      () =>
+        capabilityStore.filesArchivers || [
+          {
+            enabled: true,
+            version: '1.0.0',
+            formats: ['tar', 'zip'],
+            archiver_url: `${configStore.serverUrl}index.php/apps/files/ajax/download.php`
```

Comment:
I know that this is not your code and you just moved it around, but could you use `urlJoin` here for joining the serverUrl and the path string that starts with `index.php`?